src/Transport/HttpTransport.php

```diff
@@ -108,25 +119,26 @@ public function send(Event $event): ?string
         } else {
             try {
                 $promise->wait();
-            } catch (\Exception $exception) {
-                return null;
+            } catch (\Throwable $exception) {
+                return $event->getId();
```

Comment:
optionally the return can be removed and let it fall through to the final return